scss/util/_color.scss

```diff
@@ -59,7 +59,7 @@
 /// @param {Color} $color2 - Second color to compare.
 ///
 /// @returns {Number} The contrast ratio of the compared colors.
-@function color-pick-contrast($base, $colors: ($white, $black), $tolerance: 0) {
+@function color-pick-contrast($base, $colors: ($white, $black), $tolerance: $global-color-pick-contrast-tolerance) {
```

Comment:
Great. 
BTW.... Next time please tell this type of explanation in the  :arrow_double_up: above description! 
{{ Please dont leave description blank. It should tell what the code is doing. }}